org.eclipse.wildwebdeveloper.embedder.node/src/org/eclipse/wildwebdeveloper/embedder/node/InitializeLaunchConfigurations.java

```diff
@@ -70,13 +102,12 @@ public static String getNodeJsLocation() {
 		return null;
 	}
 
-	public static String which(String program) {
-
+	public static File which(String program) {
```

Comment:
Should we make `which()` to search for the specified program in the embedded NodeJS executable installation folder first, then using other possible paths?